conf/auth.conf

```diff
@@ -57,6 +57,27 @@
 # (ie exactly as if auth yes was present).
 #
 
+# CONTROLLING FILE ACCESS (previously in fileserver.conf)
+
+# In previous versions of Puppet, you controlled file access by adding
+# rules to fileserver.conf. In Puppet 5 with Puppet Server, you can control
+# file access in auth.conf by controlling the /file_metadata(s)/<mount point>
+# and /file_content(s)/<mount point> paths. See the Puppet Server
+# documentation at
+# https://docs.puppet.com/puppetserver/latest/config_file_auth.html.
```

Comment:
Although I guess I was just thinking about this section from the perspective of being instructions about how to replace prior usages of fileserver.conf access rules with auth.conf ones.  Mentioning `static_file_content` in the auth.conf file still seems valid irrespective of that.  I could just modify this to something like:

> In Puppet 5, you can control file access in auth.conf by controlling the /file_metadata(s)/\<mount point\>/\<file\>, /file_content(s)/\<mount point\>/\<file\>, and /static_file_content/\<file\> paths.